app/src/main/java/org/wikipedia/feed/news/NewsItemView.kt

```diff
@@ -50,6 +51,7 @@ class NewsItemView(context: Context) : WikiCardView(context) {
         newsItem.thumb()?.let {
             binding.horizontalScrollListItemImage.visibility = VISIBLE
             binding.horizontalScrollListItemImage.loadImage(it)
+            ImageZoomHelper.setViewZoomable(binding.horizontalScrollListItemImage)
```

Comment:
It might be helpful to add the same kind of `onLongClickListener` override logic as we have in  `FeaturedArticleView` and `OnThisDayCardView`, so the the long-press is consumed if the user is in the middle of zooming the image.

Reply:
Makes sense. Done.